### PR TITLE
Allow StackBlur to be applied to imageData directly (RGB and RGBA)

### PR DESCRIPTION
**Pull request by WebSeed**

### CODE REVIEW

File: src/stackblur.js [hunk 1]
```diff
@@ -630,9 +615,10 @@ function BlurStack()
...
 
-
 module.exports = {
-    image: stackBlurImage,
-    canvasRGBA: stackBlurCanvasRGBA,
-    canvasRGB: stackBlurCanvasRGB
+    image: processImage,
+    canvasRGBA: processCanvasRGBA,
+    canvasRGB: processCanvasRGB,
+    imageDataRGBA: processImageDataRGBA,
```
Comment (WebSeed) [inline src/stackblur.js]: New exports

File: src/stackblur.js [hunk 2]
```diff
@@ -630,9 +615,10 @@ function BlurStack()
...
+    imageDataRGB: processImageDataRGB
```
Comment (WebSeed) [inline src/stackblur.js]: New exports

**Comment by flozz** [comment]
Thank you for your PR, I will check it soon and merge it if there is no issue. 

Can you add documentation about your two new functions in the `README.md`?

**Comment by WebSeed** [comment]
Thank you @flozz

**Pull request merged**